docs/reference/installation/operating_systems.cue

```diff
@@ -4,7 +4,7 @@ installation: {
 	#OperatingSystem: {
 		description: string
 		family:      #OperatingSystemFamily
-		interfaces: [installation.#Interface & {_shell: shell}, ...installation.#Interface & {_shell: shell}]
```

Comment:
Picked up this tip that you don't need to set the starting element type too.

Reply:
Right, but the previous version requires at least one element.

Reply:
Aha, I see. I didn't realize that, I'll re-add.